packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -162,6 +149,27 @@ export class W3mAccountDefaultWidget extends LitElement {
     `
   }
 
+  private activityTemplate() {
+    const isSolana =
+      ChainController.state.activeChain === ConstantsUtil.CHAIN_NAME.SOLANA.toLocaleLowerCase()
+
+    return html`
+      <wui-list-item
+        iconVariant="blue"
+        icon="clock"
+        iconSize="sm"
+        ?chevron=${!isSolana}
+        disabled=${isSolana}
+        @click=${this.onTransactions.bind(this)}
+      >
+        <wui-text variant="paragraph-500" color=${isSolana ? 'gray-glass-020' : 'fg-100'}
```

Comment:
As we discussed in the Slack thread and confirmed with the designer, the `wui-text` should have `50%` opacity rather than `gray-glass-020`. We should pass a `disabled` parameter to `wui-text` and add related styling on the CSS side.

```suggestion
        <wui-text variant="paragraph-500" disabled=${isSolana}
```

Then in the wui-text styles:

```css
:host([disabled]) {
 opacity: 0.5;
}
```

should work (double check please, styling code is example)